tests/test_pytypes.py

```diff
@@ -102,11 +102,25 @@ def __repr__(self):
 
     assert m.str_from_object(A()) == "this is a str"
     assert m.repr_from_object(A()) == "this is a repr"
+    assert m.str_from_handle(A()) == "this is a str"
 
     s1, s2 = m.str_format()
     assert s1 == "1 + 2 = 3"
     assert s1 == s2
 
+    malformed_utf8 = b"\x80"
+    if sys.version_info.major < 3:
```

Comment:
TODO: Replace with `PY2`

Also, this is quite weird behavior (i.e., quite different between Python 2 and 3), but it's the same that's now already there.

Reply:
Done.